index.html

```diff
@@ -0,0 +1,145 @@
+      <div class="col">
+        <p><strong>COFFEE EVERYWHERE</strong> Now turn around and look a mile in the other direction. You see a similar coffee cup, at exactly the same temperature. Coincidence? Maybe. But if you see a similar cup in every direction, you might want to look for another explanation.</p>
+      </div>
+    </div>
+
+    <div class="inflation-row small-black-txt p-relative top-minus-250">
+      <div class="col">
+        <p><strong>THE UNIVERSE</strong> is just under 14 billion years old. From our position in the Milky Way galaxy, we can observe a sphere that is now about 92 billion light-years across. But there's a mystery. Wherever we look, the universe has an even temperature.</p>
+      </div>
+      <div class="col">
+        <p><strong>NOT ENOUGH TIME</strong> The universe is not old enough for light to have traveled the vast distance from one side of the universe to the other, and there has not been enough time for scattered patches of hot and cold to mix into an even temperature.</p>
+      </div>
+      <div class="col">
+        <p><strong>DISTANT COFFEE</strong> At a smaller scale, imagine using a telescope to look a mile in one direction. You see a coffee cup, and from the amount of steam, you can estimate its temperature and how much it has cooled.</p>
+      </div>
+      <div class="col">
+        <p><strong>COFFEE EVERYWHERE</strong> Now turn around and look a mile in the other direction. You see a similar coffee cup, at exactly the same temperature. Coincidence? Maybe. But if you see a similar cup in every direction, you might want to look for another explanation.</p>
+      </div>
+    </div>
+
+
+   </div>
+
+    <div class="story-bloc my-4">
+
+
+    <p>Dr. Kovac said the chance that the results were a fluke was only one in 10 million.
+
+    <p>Dr. Guth, now 67, pronounced himself “bowled over,” saying he had not expected such a definite confirmation in his lifetime.
+
+
+    <p>“With nature, you have to be lucky,” he said. “Apparently we have been lucky.”</p>
+
+    <p>The results are the closely guarded distillation of three years’ worth of observations and analysis. Eschewing email for fear of a leak, Dr. Kovac personally delivered drafts of his work to a select few, meeting with Dr. Guth, who is now a professor at Massachusetts Institute of Technology (as is his son, Larry, who was sleeping that night in 1979), in his office last week.</p>
+
+    <p>“It was a very special moment, and one we took very seriously as scientists,” said Dr. Kovac, who chose his words as carefully as he tended his radio telescopes.</p>
+
+    <p>Andrei Linde of Stanford, a prolific theorist who first described the most popular variant of inflation, known as chaotic inflation, in 1983, was about to go on vacation in the Caribbean last week when Chao-Lin Kuo, a Stanford colleague and a member of Dr. Kovac’s team, knocked on his door with a bottle of Champagne to tell him the news.</p>
+   </div>
+
+
+ </div>
+ </div>
+
+
```

Comment:
`REQUIRED`

The page is currently missing some section. Please check the original site.